common/test/acceptance/tests/studio/test_studio_outline.py

```diff
@@ -152,11 +156,12 @@ class PublishState(object):
     class UnitState(object):
         """ Represents the state of a unit """
 
-        def __init__(self, is_released, publish_state, is_locked):
+        def __init__(self, is_released, publish_state, is_locked, is_restricted):
```

Comment:
Nit: you could make "is_restricted" have a default value (False) so that you don't have to update all the existing callers of the method.